Lib/test/test_ast.py

```diff
@@ -216,6 +216,10 @@ def to_tuple(t):
 
 ]
 
+extra_validation_snippets = [
```

Comment:
Why not add it to `exec_tests` or `single_tests`?

Reply:
I was afraid that the diff will be very difficult to read because the regenerated `exec_results`, but it turned out to be very simple. I have added the test to `exec_results` instead of creating a new `extra_validation_snippets`.